[IMP] website: cookies bar (#9571)
odoo/documentation: content/applications/websites/website/configuration/cookies_bar.rst

Cookies bar
===========

**Cookies** are small text files sent to your device when you visit a website. They remember user
information like login details, preferences, and browsing history. **Essential cookies** are
necessary for the website to function, while **optional cookies** analyze behavior or display ads.

Data protection laws require notifying users about data collection methods and purposes.
**Cookie bars** fulfill this obligation by informing users on their first visit and allowing them to

> decide whether to store non-essential cookies on their devices.

Author: Good point. I will use optional because it is the term used in Odoo.

Configuration
=============

To enable the cookies bar on your website, go to :menuselection:`Website --> Configuration -->
Settings` and enable :guilabel:`Cookies Bar` in the :guilabel:`Privacy` section.

.. note::
   The :ref:`Cookie Policy <cookies-bar/policy>` page (/cookie-policy) is automatically created when
   you enable the cookies bar.

.. _cookies-bar/customization:

Customization
=============

From your website, click :guilabel:`Edit`, go to the :guilabel:`Invisible Elements` section at the
bottom of the panel, and click :guilabel:`Cookies Bar`. You can select a :guilabel:`Layout`, adjust
the :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
background when the cookies bar is displayed on the screen.

Click anywhere in the building block to further customize the appearance of the cookies bar using

> :guilabel:`Block` and/or :guilabel:`Column` customization options.

Reviewer: You mention 'Block' and 'Colum', but I also see 'Inline text'. For consistency, I would also add it, unless there's a reason not to :)

Cookies bar
===========

**Cookies** are small text files sent to your device when you visit a website. They are processed
and stored by your browser and track user information like login details, preferences, and browsing
history. **Essential cookies** are necessary for the website to function, while **optional cookies**
analyze behavior or display ads.

Data protection laws require notifying users about data collection methods and purposes.
**Cookie bars** fulfill this obligation by informing users on their first visit and allowing them to
decide whether to store all or only essential cookies on their devices.

Configuration
=============

To enable the cookies bar on your website, go to :menuselection:`Website --> Configuration -->
Settings` and enable :guilabel:`Cookies Bar` in the :guilabel:`Privacy` section.

.. note::
   The :ref:`Cookie Policy <cookies-bar/policy>` page (/cookie-policy) is automatically created when
   you enable the cookies bar.

.. _cookies-bar/customization:

Customization
=============

> From your website, click :guilabel:`Edit`, go to the :guilabel:`Invisible Elements` section at the

Reviewer 2: ```suggestion
To adapt the display of the cookie bar, click :guilabel:`Edit`, go to the :guilabel:`Invisible Elements` section at the
```
I think it's still useful to provide context for what we're going to explain in the next paragraph instead of diving right into giving instructions. (You could use *appearance* instead of *display* if you prefer)

> bottom of the panel, and click :guilabel:`Cookies Bar`. You can select a :guilabel:`Layout`, adjust
> the :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
> background when the cookies bar is displayed on the screen.

Reviewer 2: ```suggestion
bottom of the panel, and click :guilabel:`Cookies Bar`. You can modify the :guilabel:`Layout` and :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
background when the cookies bar is displayed on the screen.
```
- I prefer "modify the layout" since the Layout field is not empty by default (you could say "select another :guilabel:`Layout` to be more exact, but this sounds very bof to me)
- *Modify* is the same as *adjust* in this context, so I suggest removing *adjust* to simplify and shorten the sentence a little bit.

Author: I've chosen not to describe each type of layout because, in my opinion, it doesn't add value to the documentation. Users will try out the different layouts and see that they can change the position if they select Pop-up. I aimed to be more concise.

Click anywhere in the building block to further customize the appearance of the cookies bar using
:guilabel:`Block`, :guilabel:`Column` and/or :guilabel:`Inline Text` customization options.

To edit the contents of the cookies bar (i.e., the consent message), click directly in the building
block.

.. _cookies-bar/policy:

Cookie policy
=============

When you enable the cookies bar for your website, Odoo creates the **Cookie Policy** page
(`/cookie-policy`) containing a non-exhaustive list of cookies, with their purpose and examples. To
access it, click the :guilabel:`Cookie Policy` hyperlink in the cookies bar or open the page from

> :menuselection:`Website --> Site --> Pages`.

Reviewer 2: This is totally optional, but maybe we could add a :doc: link to the Pages documentation (maybe on "open the page")?

> You can edit the content of the page as per your needs by clicking the :guilabel:`Edit` button.

Reviewer 2: ```suggestion
To adapt the content of the page according to your needs, click the :guilabel:`Edit` button.
```
- *adapt*: to avoid repeating *edit*
- "as per your needs" sounds a little bit too formal and old-fashioned to me
- In general (might be a few exceptions of course), instead of saying "You can do xxx *by doing* yyy", we can use a more direct wording and say: "to do xxx, do yyy"

.. tip::

>    Add a link to this page in your website's footer, for example.

Reviewer 2: ```suggestion
   You could add a link to this page in your website's footer, for example.
```
I this case, since we're really *suggesting* something and not *giving instructions*, I think it might sound a little "softer" to keep "You could"
